packages/gg_images/PKGBUILD

```diff
@@ -0,0 +1,37 @@
+pkgname=gg_images
```

Comment:
Please follow the packaging style. Do not skip [the comments](https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/PKGBUILD-go#L1)